pkg/sessionctx/stmtctx/stmtctx.go

```diff
@@ -1517,3 +1559,21 @@ func GetStmtLabel(ctx context.Context, node ast.StmtNode) string {
 	}
 	return ast.GetStmtLabel(node)
 }
+
+// TableDelta stands for the changed count for one table or partition.
+type TableDelta struct {
+	Delta    int64
+	Count    int64
+	InitTime time.Time // InitTime is the time that this delta is generated.
+	TableID  int64
+}
+
+// Clone returns a cloned TableDelta.
+func (td TableDelta) Clone() TableDelta {
+	return TableDelta{
```

Comment:
can directly return `td`, or let caller do 

```go
tdCloned := td
```

because the struct contains no pointer

Reply:
I don't want to change this since I only move it from `pkg/sessionctx/variable/session.go` to here.